### PR TITLE
Fixed sleepers and buckle duplication

### PR DESCRIPTION
**Pull request by NanakoAC**
Fixed a bug where buckled people could be placed into multiple machines
simultaneously

Medical machines will now unbuckle a patient before accepting them.

Fixed dialysis beaker being ejected from sleeper beds when you eject the patient

Dialysis beaker can now be checked and ejected while a sleeper is empty

### CODE REVIEW

**Comment by NanakoAC** [comment]
fixes #344

File: Nanako-PR-345.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#################################
+
+# Your name.  
+author: Nanako
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: False
```
Comment (skull132) [inline Nanako-PR-345.yml]: Set to true. This is _always_ true.

File: code/game/machinery/adv_med.dm [hunk 2]
```diff
@@ -103,13 +103,18 @@
...
 	if (src.occupant)
 		user << "\blue <B>The scanner is already occupied!</B>"
 		return
 	if (M.abiotic())
 		user << "\blue <B>Subject cannot have abiotic items on.</B>"
 		return
+
+	if (M.buckled && istype(M.buckled, /obj/structure))//We must make sure the person is unbuckled before they go in
+		var/obj/structure/LB = M.buckled
+		LB.user_unbuckle_mob(user)
```
Comment (skull132) [inline code/game/machinery/adv_med.dm]: This is actually DNM atm, I should have noted this earlier. Coding principle for SS13 is: the game never does things like this for you. Notice how in all of the lines above, it gives the user feedback and returns, demanding that the user fix the error.

By having the game automatically unbuckle the dude, this principle is broken. Instead, do something like:
```
user << "<span class='warning'>You cannot do that! [M] is buckled to [M.buckled]!</span>"
return
```

**Comment by skull132** [comment]
I need to pay more attention. Fix the thing I commented, and I'll merge it whenever I up us to 510 later this week.

**Pull request merged**